mirrord/sip/src/whitespace.rs

```diff
@@ -0,0 +1,67 @@
+// source: https://github.com/dtolnay/syn/blob/master/src/whitespace.rs
```

Comment:
include its license

Reply:
Ok, like commit their license file to our repo?
Also apparently we can choose between MIT and Apache 2.0,  so MIT, right?

Reply:
not as a file, just as a header to this file.

Reply:
Added.